kie-bom/pom.xml

```diff
@@ -1213,6 +1213,13 @@
         <version>${version.org.kie}</version>
         <classifier>sources</classifier>
       </dependency>
+      <dependency>
+        <groupId>org.kie.workbench.screens</groupId>
+        <artifactId>kie-wb-common-data-modeller-backend</artifactId>
+        <version>${version.org.kie}</version>
+        <type>test-jar</type>
+        <scope>test</scope>
```

Comment:
This is BOM pom. The purpose of `dependencyManagement` is just to define dependency version for all project submodules. Scope must be defined in dependencies section of individual depending on this bom.

In other words just follow [these instructions](https://github.com/jsoltes/droolsjbpm-build-bootstrap/blob/9a527fb5df710ad9d90f8adb7c443362421e7a2f/kie-bom/pom.xml#L32-L34) and remove the `scope` element :-)